### ISSUE TITLE
Lack of proper documentation

### ISSUE THREAD

**Issue by fpiraneo** [ceph/ceph-ansible#6828]
**Is your feature request related to a problem? Please describe.**
Unable to install on debian / CentOS with actual procedures.

**Describe the solution you'd like**
Step by step procedure to deploy a fully working cluster on both debian and CentOS.

**Describe alternatives you've considered**
Step by step procedure to manually deploy a fully working cluster on both debian and CentOS.

The actual documentation is fairly incomplete and outdated and cover the installation only on RedHat Enterprise.

**I already offered my help to write such procedures but I need support from the team.**

**Comment by devinjdawson** [comment]
I have to agree with this assessment. This current release of arm64 ceph is not even compatible with the majority of people trying to use it on Ubuntu. I have been trying to get this application set up for months, and it is mostly boils down to lack of proper documentation and packages not being updated. I'm sure this works on older RHEL or CENT x86 without much issue, but just following along with the basic information available can be misleading and ends up wasting more time than necessary for simple things. The project certainly could use some help.

For example: There were no current arm64 focal packages 2 months ago, so I had to update my entire operating system to hirsute to try to get it to work. That being said, ceph doesn't actively update hirsute releases, and the 16 .2.4 release does not actually work with arm64. The only package that works that is currently made available that should work is Focal  16.2.5, but my entire operating system is now on hirsute. 

On top of this, the release key that ceph uses is not compatible with ubuntu/apt/ansible,  So either I have to manually install all of the packages on each machine (giving that the documentation in site.yml is incomplete) or downgrade all of my servers that I specifically updated to an unsupported releases because that was the only version of ceph available at the time.

I've wasted more time trying to install this application than ever even trying to get it to work properly. One may criticize my ability all they want, but it shouldn't be this much of an enigma to get a simple application to run. In reality, the application is not an issue, it just not properly supported.

**Comment by fpiraneo** [comment]
I'm developing my own procedures to manually bootstrap a ceph cluster; is not so much complicated and I think that using ansible may conceal know how useful when running the cluster and you get in throuble.
The worse however is yet to come: I'm running through developing php scripts to interact with the cluster with librados and the whole radosphp documentation is: "Read the f***g code!" :-D

**Comment by RubenGarcia** [comment]
I confirm that the ansible scripts for stable-4.0 fail on Debian 10.

**Comment by RubenGarcia** [comment]
Also, your bot "This issue has been automatically marked as stale" needs to be configured to give some more time for people to check the issues. It seems to be closing any issues without providing a response, workaround or fix.

**Comment by SLongofono** [comment]
Stable 4.0 and 5.0 failing for Red Hat/Centos 7 and 8 due to the release keys
